src/main/java/com/algorand/algosdk/v2/client/Utils.java

```diff
@@ -1,19 +1,179 @@
 package com.algorand.algosdk.v2.client;
 
+import java.math.BigInteger;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+import com.algorand.algosdk.crypto.Address;
+import com.algorand.algosdk.transaction.SignedTransaction;
+import com.algorand.algosdk.transaction.Transaction;
+import com.algorand.algosdk.transaction.Transaction.Type;
 import com.algorand.algosdk.v2.client.common.AlgodClient;
 import com.algorand.algosdk.v2.client.common.Response;
+import com.algorand.algosdk.v2.client.model.DryrunRequest;
+import com.algorand.algosdk.v2.client.model.Application;
+import com.algorand.algosdk.v2.client.model.ApplicationParams;
+import com.algorand.algosdk.v2.client.model.ApplicationStateSchema;
+import com.algorand.algosdk.v2.client.model.Account;
+import com.algorand.algosdk.v2.client.model.Asset;
 import com.algorand.algosdk.v2.client.model.NodeStatusResponse;
 import com.algorand.algosdk.v2.client.model.PendingTransactionResponse;
 
 public class Utils {
+
+    // https://github.com/algorand/go-algorand/blob/e466aa18d4d963868d6d15279b1c881977fa603f/libgoal/libgoal.go#L1089-L1090
+    private static Long defaultAppId = 1380011588L;
```

Comment:
nit: we can make this variable `final`:
```suggestion
    private static final Long defaultAppId = 1380011588L;
```